vendor/vendor.json

```diff
@@ -0,0 +1,75 @@
+{
+	"comment": "",
+	"ignore": "test",
+	"package": [
+		{
+			"checksumSHA1": "FUhczVSvt/VQAlWfzATo1YLez2E=",
+			"path": "github.com/Financial-Times/go-fthealth/v1a",
+			"revision": "f9074331b3ab48bca0e6d0afc211a673d88ac925",
+			"revisionTime": "2016-06-16T09:04:59Z"
+		},
+		{
+			"checksumSHA1": "yceqWuIbFzntmSHl66OwNoP/c6E=",
+			"path": "github.com/Financial-Times/message-queue-go-producer/producer",
```

Comment:
In my opinion, it is more convenient if you vendor a specific version of the libraries that you use.
In this way, who is going to make a change to the app knows what's the version of the library in use.
e.g.: `govendor sync github.com/Financial-Times/message-queue-go-producer/producer@v0.1.0`